### ISSUE TITLE
ACC-261.1 (Level AA) _ Modify Modules _ Text _ Doesn’t satisfy min contrast ratio 

### ISSUE THREAD

**Issue by shubha1090** [kyma-project/busola#3137]
1.1 Internal Message Details
Internal Message:	
Test Plan:	ACC SAP Kyma runtime 2024
Message Priority:	High
Affected UI	Text
Affected Check Point:	ACC-261.1
Short text:	ACC-261.1 (Level AA) _ Modify Modules _ Text _ Doesn’t satisfy min contrast ratio 

1.2 Application Details
Application Name	Kyma-dashboard
Technology:	SAP UI5 
Component:	
URL:	https://dashboard.stage.kyma.cloud.sap/cluster/garden-kyma-stage--c-7074971-external/kymamodules
Username/Password:	NA
1.3 Test Environment Details
Web browser:	Google Chrome Version 127.0.6533.89
Operating System:	Microsoft Windows 11 x64
Screen Reader:	JAWS 2024 Version 2024.2406.121 ILM

1.4 JIRA/SNOW Details
Issue:	Bug
Labels:	Accessibility

2 Screen Flow
•	Launch the application using given URL. 
•	Select the ‘Link to Kyma dashboard’ from the Kyma Environment section on your SAP BTP subaccount page. Now Select ‘Modify Modules’ button which opens ‘Modules’ page.
•	Download and open Colour Contrast Analyser tool and check for the contrast ratios of  ‘Ready’ object status text from the main region.

3. Issue Type: Manual
Observed Behavior:
The ‘Ready’ object status text across the application which has Green background and white text doesn’t satisfy the minimum contrast ratio of 4.5:1 in default Light theme. 
Screenshot:
 
Case 2:
 

Expected Behavior:
The ‘Ready’ object status text across the application which has Green background and white text must satisfy the minimum contrast ratio of 4.5:1 in default Light theme. A minimum contrast improves legibility for partially sighted users (for example, users with low visual acuity or color vision deficiencies) and makes content easier to use. Therefore, all visual details (such as lines, colors, …) that identify a UI element, show its state or indicate its interoperability need to fulfill a minimum contrast.
Note: 
This issue exists across the application and thus fix should be provided for all the occurrences.

________________________________________________________________________
Kindly refer to attached error and issue processing document for more details.
Regards,
Shubha

**Comment by shubha1090** [comment]
[ACC-261.1 (Level AA) _ Modify Modules _ Text _ Doesn’t satisfy min contrast ratio.docx](https://github.com/user-attachments/files/16659566/ACC-261.1.Level.AA._.Modify.Modules._.Text._.Doesn.t.satisfy.min.contrast.ratio.docx)

[Issue processing-09.2021 (002).docx](https://github.com/user-attachments/files/16659568/Issue.processing-09.2021.002.docx)

**Comment by mrCherry97** [comment]
We are using UI5, and staying with UI5/Fiori guidelines
